NOTICE

```diff
@@ -1,2 +1,2 @@
-Copyright (c) 2021 Pravega Authors.
-Copyright (c) 2017-2021 Dell Inc., or its subsidiaries. All Rights Reserved.
\ No newline at end of file
+Copyright (c) 2021-2022 Pravega Authors.
+Copyright (c) 2017-2022 Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
It should still keep `2021` this line because dell copyrights are just used in the past. @derekm Please confirm if I am wrong here.

Reply:
Yes, that's correct @crazyzhou, the Dell Inc. copyright doesn't need to be maintained any longer. "Pravega Authors" is the plural copyright holder under CNCF.